packages/eas-cli/src/commands/branch/list.ts

```diff
@@ -18,6 +18,59 @@ import { findProjectRootAsync, getProjectFullNameAsync } from '../../project/pro
 import { getActorDisplayName } from '../../user/actions';
 
 const BRANCHES_LIMIT = 10_000;
+export async function listBranchesAsync({
```

Comment:
```suggestion

async function listBranchesAsync({
```
I think this was unintentionally exported from this modules?

Reply:
Good catch, I am using it now: https://github.com/expo/eas-cli/pull/253/files#diff-2fb981f3a86534f69bea6c5560ce229c420628a9969ee0a07169d7623a21b1e0R139